jupyter_rfb/_utils.py

```diff
@@ -0,0 +1,93 @@
+import io
+import builtins
+import traceback
+from base64 import encodebytes
+
+from IPython.display import DisplayObject
+import ipywidgets
+
+from ._png import array2png
+
+_original_print = builtins.print
+
+
+class RFBOutputContext(ipywidgets.Output):
+    """An output widget with a different implementation of the context manager.
+
+    Handles prints and errors in a more reliable way, that is also
+    lightweight (i.e. no peformance cost).
+
+    See https://github.com/vispy/jupyter_rfb/issues/35
+    """
+
+    capture_print = False
+    _prev_print = None
+
+    def print(self, *args, **kwargs):
+        """Print function that show up in the output."""
+        f = io.StringIO()
+        kwargs.pop("file", None)
+        _original_print(*args, file=f, flush=True, **kwargs)
```

Comment:
I don't understand why this should capture from any `print`, regardless of `file` argument? Unless it's `sys.stdout` or `sys.stderr`, it's going to break things in pretty weird ways.

Reply:
Oh good point! It should probably check the file argument and only consume if it looks like standard output ... I will make an issue.

Reply:
#83